src/main/scala/beam/sim/BeamMobsim.scala

```diff
@@ -204,7 +204,7 @@ class BeamMobsim @Inject() (
         case VehicleCategory.Bike => BeamMode.BIKE
       }.toList
 
-      val cavs = vehicles.filter(_.beamVehicleType.isCav).toList
+      val cavs = vehicles.filter(_.isCAV).toList
```

Comment:
Should we rename all cav to connectedautomationvehicle as used elsewhere?

Reply:
Whoever does not understand what CAV means, can see the meaning here: 
`  def isCAV: Boolean = beamVehicleType.isConnectedAutomatedVehicle`
There is not need to replace it elsewhere